Resources/Prototypes/_Shitmed/Recipes/Lathes/Packs/robotics.yml

```diff
@@ -1,18 +1,23 @@
 ## Static
 
 - type: latheRecipePack
-  id: IPCPartsStatic
+  id: IPCLimbsStatic # DeltaV - Share Surgery Parts between Epi and Med.
   recipes:
-  - TorsoIPC
   - LeftArmIPC
   - RightArmIPC
   - LeftLegIPC
   - RightLegIPC
-  - HeadIPC
   - LeftHandIPC
   - RightHandIPC
   - LeftFootIPC
   - RightFootIPC
+
```

Comment:
Use a block comment to show all the code that is DeltaV (Not just in the parent)
`# Begin DeltaV Content`
`# End DeltaV Content`